dataland-frontend/src/components/forms/parts/fields/MultiSelectFormFieldBindData.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div class="form-field" :data-test="name">
+    <UploadFormHeader :label="label" :description="description" :is-required="required" />
+    <MultiSelectFormElementBindData
```

Comment:
As I have already outlined for the file `MultiSelectFormElementBindData.vue`,  I am not sure if we even need this special component.
Why can't we use the `MultiSelectFormField.vue` and set the "ignore" to "true" ?

Reply:
as above

Reply:
Yes.  I will close this conversation since we are already discussing this in another conversation.